test/test_zany_mapping.py

```diff
@@ -86,9 +83,10 @@ def test_C1_elements(ref_cell, phys_cell, element):
 
 
 @pytest.mark.parametrize("element, degree", [
-                         *((finat.Argyris, k) for k in range(5, 8)),
-                         *((finat.HsiehCloughTocher, k) for k in range(3, 6))
-                         ])
+    (finat.QuadraticPowellSabin6, 2), (finat.QuadraticPowellSabin12, 2),
```

Comment:
These should be tested with `test_C1_elements`

Reply:
The are in the higher order C1 tests below since I wanted to be able to set `avg=True`.  Maybe we should split up the tests?

Reply:
Wait, I can do this with finat-level kwargs in the regular C1 test.  I'll move them there since Powell-Sabin certainly isn't higher-order.